src/lib/entities/model.py

```diff
@@ -45,4 +45,14 @@ class Entity:  # pylint: disable=R0902
     dns_names: List[str]
 
 
-ExtractEntitesFunc = Callable[[MetricsContext, str, GCPService], Iterable[Entity]]
+ExtractEntitiesFunc = Callable[[MetricsContext, str, GCPService], Iterable[Entity]]
+
+
+class EntitiesExtractorData:
```

Comment:
(minor) you could use the same annotation as above (@dataclass) -> it makes creating classes easier when a class mostly contains attributes

Reply:
Done